src/clj_puppetdb/http.clj

```diff
@@ -16,13 +17,14 @@
 
 (defn- make-https-client
   [^String host {:keys [ssl-ca-cert ssl-cert ssl-key vcr-dir]}]
-  {:pre  [(every? file? [ssl-ca-cert ssl-cert ssl-key])
+  {:pre  [(or (every? file? [ssl-ca-cert ssl-cert ssl-key])
+              (throw (IllegalArgumentException.
+                       (format "Any of the following files does not exist: ssl-ca-cert=%s, ssl-cert=%s, ssl-key=%s"
+                               ssl-ca-cert ssl-cert ssl-key))))
```

Comment:
I'd suggest improving cert files checking such that the error message states the file that is missing, something in this vein: 
[^String host opts]
  {:pre  [(every? #(or (file? (get opts %)) (throw (IllegalArgumentException.
                       (str "The following files does not exist: " (name %) "=" (get opts %)))))
                               [:ssl-ca-cert :ssl-cert :ssl-key])]